rest/index.py

```diff
@@ -178,6 +178,16 @@ def ipintel():
     return response
 
 
+@route('/ipsearch', methods=['POST'])
+@route('/ipsearch/', methods=['POST'])
+def ipsearch():
```

Comment:
We don't need to define this route, meteor handles the opening of the url so this is never actually called.

Reply:
hmm, I think  you are right, originally I was using this section to open the url, but later moved it.
I'll update :) ty!